tests/integration/test_analysisd/test_eps/test_stop_processing_events.py

```diff
@@ -0,0 +1,94 @@
+import os
+from time import sleep
+from datetime import datetime
+import pytest
+
+from wazuh_testing.tools.configuration import load_configuration_template, get_test_cases_data, \
+                                              get_simulate_agent_configuration
+from wazuh_testing.modules.eps import event_monitor as evm
+from wazuh_testing.modules.eps import PERCENTAGE_PROCESS_MSGS, QUEUE_SIZE
+
+
+pytestmark = [pytest.mark.server]
+
+# Reference paths
+TEST_DATA_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+CONFIGURATIONS_PATH = os.path.join(TEST_DATA_PATH, 'configuration_template')
+TEST_CASES_PATH = os.path.join(TEST_DATA_PATH, 'test_cases')
+
+# Configuration and cases data
+configurations_path = os.path.join(CONFIGURATIONS_PATH, 'configuration_stop_processing_events.yaml')
+configurations_simulate_agent_path = os.path.join(TEST_DATA_PATH,
+                                                  'configuration_simulate_agent.yaml')
+t1_cases_path = os.path.join(TEST_CASES_PATH, 'cases_stop_process_events.yaml')
+
+# Stop processing events test configurations (t1)
+t1_configuration_parameters, t1_configuration_metadata, t1_case_ids = get_test_cases_data(t1_cases_path)
+t1_configurations = load_configuration_template(configurations_path, t1_configuration_parameters,
+                                                t1_configuration_metadata)
+
+# Get simulate agent configurations (t1)
+params_stop_processing_events = get_simulate_agent_configuration(configurations_simulate_agent_path)
+maximum_eps = [metadata['maximum'] for metadata in t1_configuration_metadata]
+timeframe_eps_t1 = [metadata['timeframe'] for metadata in t1_configuration_metadata]
+total_msg = maximum_eps[0] * timeframe_eps_t1[0] * 3
+if total_msg > QUEUE_SIZE:
+    total_msg = QUEUE_SIZE - 1
```

Comment:
Why this conditional?

Reply:
Done [here](https://github.com/wazuh/wazuh-qa/pull/3181/commits/f817116435639fa720611bd68eb90ced7c606062)